### PR TITLE
Add Invalid CIDR message

### PR DESCRIPTION
**Pull request by rogbas**
Adding new Service Log to communicate invalid CIDR configuration

### CODE REVIEW

**Comment by billmvt** [comment]
@wgordon17 looks like a good message to me, interested in your +1 as well

File: osd/InvalidCIDR.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Error",
+    "service_name": "SREManualAction",
+    "cluster_uuid": "${CLUSTER_UUID}",
+    "summary": "Installation blocked, action required",
+    "description": "Your cluster's installation is blocked on invalid CIDR configurations. Please review the Node/Service/Pod CIDRs configuration specified during cluster provisioning and try again. For more information please refer to: https://docs.openshift.com/dedicated/4/getting_started/accessing-your-services.html#dedicated-creating-your-cluster_access",
```
Comment (aweiteka) [inline osd/InvalidCIDR.json:6]: > ...and try again

Does that mean the customer needs to delete this cluster and create a new one? Or will installation re-try when values are updated? I think it's the former so we should spell that out clearly for the customer.

**Comment by wgordon17** [comment]
Recommended rewording

```
"summary": "Installation has failed",
"description": "Your cluster's installation has failed related to an invalid CIDR configuration. The {name of CIDR} range is invalid and causing an error '{Error message from uninstaller}'. You must delete this cluster, and recreate with {Recommended fix}.",
```

Example name of CIDR
> Machine CIDR

Example error message
> subnet's CIDR range start 10.1.100.0 is outside of the specified machine networks

Example recommended fix
> a larger Machine CIDR that includes all of the subnets CIDR ranges

File: osd/InvalidCIDR.json (hunk 1 above)
Comment (fahlmant) [reply osd/InvalidCIDR.json:6]: Its the former.

**Comment by rogbas** [comment]
@cattias @billmvt  Updated with Will's suggestions, ready for review/merge.

**Pull request merged**